components/html-block/html-block.js

```diff
@@ -163,7 +163,7 @@ class HtmlBlock extends RtlMixin(LitElement) {
 				display: none;
 			}
 			:host([inline]),
-			:host([inline]) div.d2l-html-block-rendered {
+			:host([inline]:not([no-deferred-rendering])) div.d2l-html-block-rendered {
```

Comment:
Swapping this inline CSS block with the hidden case also solves this problem, but I personally like the more explicit rules.

Reply:
Hmm, but we probably also need another exception on the host, so perhaps it would indeed just be cleaner to rely on implicit ordering. Certainly open to other thoughts on that.

Reply:
I try order the selectors to avoid `:not(...)` since I find it _usually_ results in simpler CSS. I use it, but sparingly when necessary or when the alternative is difficult for the reader to understand. The `:not(...)` represents an extra selector check the browser needs to do. Whether the browser has optimized the matching to include negating selectors I am unsure.

Reply:
Yeah, I have it restructured to rely on the CSS order in my simplification work, so I'll align that here.